ZenPacks/zenoss/Microsoft/Windows/objects/objects.xml

```diff
@@ -16782,9 +16782,40 @@ True
 </object>
 <!-- ('', 'zport', 'dmd', 'Events', 'Status', 'Winrm') -->
 <object id='/zport/dmd/Events/Status/Winrm' module='Products.ZenEvents.EventClass' class='EventClass'>
+<object id='Ping' module='Products.ZenEvents.EventClass' class='EventClass'>
+</object>
+<object id='Auth' module='Products.ZenEvents.EventClass' class='EventClass'>
+<object id='PasswordExpired' module='Products.ZenEvents.EventClass' class='EventClass'>
+<tomanycont id='instances'>
+<object id='Password Expired Default' module='Products.ZenEvents.EventClassInst' class='EventClassInst'>
+<property type="string" id="eventClassKey" mode="w" >
+defaultmapping
+</property>
+<property type="int" id="sequence" mode="w" >
+1001
+</property>
+<property type="string" id="rule" mode="w" >
+(getattr(evt, 'eventClassKey', '') or '').startswith('MW|PasswordExpired')
+</property>
+</object>
+</tomanycont>
+</object>
+<object id='WrongCredentials' module='Products.ZenEvents.EventClass' class='EventClass'>
+<tomanycont id='instances'>
+<object id='Wrong Credentials Default' module='Products.ZenEvents.EventClassInst' class='EventClassInst'>
+<property type="string" id="eventClassKey" mode="w" >
+defaultmapping
```

Comment:
This should be _MW|WrongCredentials_ instead of _defaultmapping_. Using defaultmapping is more costly when processing events, and should only be used when necessary.